README.md

```diff
@@ -15,6 +15,7 @@ pickup and staff tooling with other systems.
 - [Orders](/doc/api_v1_orders.md)
 - [Webhooks](/doc/webhooks.md)
 - [iOS SDK](https://github.com/RadiusNetworks/flybuy-ios)
+- [White-Label Domains](/doc/white_label_domains)
```

Comment:
The other links have the `.md` extension, do we not need it?

Reply:
Good catch, I'll add it to be consistent 👍